### PR TITLE
#157310819 Team profile page

### PR DESCRIPTION
**Pull request by chuzksy-codeactive**
#### What does this PR do?
Users should be able to see the profiles of the team members.
#### Description of Task to be completed?
Users should be able to see the pictures, bios and social media links of the members of the software development team.
#### How should this be manually tested?
Clone the app and install dependencies  by running `npm install`
Then run `npm run start`
Go to `http:localhost:4000/#!/team`

### CODE REVIEW

File: public/views/team.html [hunk 1]
```diff
@@ -0,0 +1,272 @@
...
+          </div>
+        </div>
+      </div>
+      <div class="col s4 member">
+        <img src="/img/emma.png" alt="">
+        <h6>Emmanuel Abaye</h6>
+        <p>Software Developer D0 (SIMs)</p>
+        <p class="ajust-text">Emmanuel Abaye is a D0 software developer at Andela. He currently works with with Javascript technologies. Emmanuel loves
+        learning and sharing knowledge. If he’s not coding, you can catch him unwinding by hanging out with friends and watching
+        movies./p>
```
Comment (emmabaye) [inline public/views/team.html]: This closing p tag is missing an opening bracket.

**Review by emmabaye — COMMENTED** [review]
Great work, however, I think the height of each team member profile should be even. Also, the top of the page is not properly responsive. The user details for product owner and TTL doesn't display properly.

**Review by amarachukwu-agbo — COMMENTED** [review]
Good job on the profile page. I have viewed the deployment on Heroku and I found that the profile pictures of some team members have a different frame. Olusola's picture has a yellow and blue frame, yours has a grey frame while the rest of the team's have a blue frame. Please make the frame uniform so that the design is consistent.

**Comment by andela-moe** [comment]
Looks good to me

**Pull request merged**